sandbox/crate/config/crate.yml

```diff
@@ -9,7 +9,7 @@ ssl.keystore_filepath: ../server/src/test/resources/keystore.pcks12 # working di
 ssl.keystore_password: keystorePassword
 ssl.keystore_key_password: keystorePassword
 
-path.repo: ../sandbox/crate/repo # working dir is ${PROJEC_DIR}/app
+path.repo: /home/matriv/crate/crate/sandbox/crate/repo # working dir is ${PROJEC_DIR}/app
```

Comment:
I guess you didn't want to commit this ;)

Reply:
thx!